installinator/src/dispatch.rs

```diff
@@ -150,6 +150,18 @@ struct InstallOpts {
     #[clap(long)]
     install_on_gimlet: bool,
 
+    //TODO(ry) this probably needs to get plumbed somewhere instead of relying
+    //on a default.
+    /// The first gimlet data link to use.
+    #[clap(long, default_value = "cxgbe0")]
```

Comment:
Plumbing things through to installinator before it has network access is pretty tricky - the only path we have right now is via the SP holding a blob of data for us, which is how we send it the sha256 hash of the artifacts it should download (a serialized [InstallinatorImageId](https://github.com/oxidecomputer/omicron/blob/60b6cc205ab3459a825e0a9d2ef69309326a3485/ipcc-key-value/src/lib.rs#L46-L64)). Are we sure defaults aren't okay here? Is it something installinator could infer from inspecting hardware instead of something it needs to be told?

Reply:
I think the defaults are ok for the product, real gimlets, and development environments that happen to have Chelsio NICs that get numbered `cxgbe0` and `cxgbe1`. If we want to run installinator on lab setups with commodity hardware that have something else like `igb0` and `igb1`, then we'd need to plumb something different in somehow - and I'm not sure how that would work.

Reply:
I'm not sure how installinator in a non-gimlet environment would work at all in the context of a trampoline image, since it fetches the `InstallinatorImageId` over IPCC which only exists on gimlets. We can (and have, for testing) run `installinator` as a standalone binary with a variety of command line args; if these defaults are ok for real gimlets I think have the command line args to override for tests puts this on equal footing with all the other non-gimlet installinator options.